assets/src/js/frontend/give-stripe-becs.js

```diff
@@ -30,6 +24,22 @@ document.addEventListener( 'DOMContentLoaded', function( e ) {
 	Array.prototype.forEach.call( formWraps, function( formWrap ) {
 		const formElement = formWrap.querySelector( '.give-form' );
 
+		// Bailout, if `form_element` is null.
```

Comment:
Why would we bail? Is this an expected scenario? Or should we throw an error? If this expected please update the comment to explain what that scenario is and why it's ok to return at this point.

Reply:
Added detailed comments as it is expected scenario